package.json

```diff
@@ -74,10 +74,10 @@
     "semver": "^6.0.0",
     "temp-sandbox": "^3.0.0",
     "typescript": "^3.4.5",
-    "webpack": "^4.32.0"
+    "webpack": "^5.11.1"
   },
   "dependencies": {
-    "@types/webpack": "^4.4.31",
+    "@types/webpack": "^4.41.25",
```

Comment:
You may want to remove this dependency since webpack 5 has it's own typescript types which conflict with these types. This is also the reason why a lot of people have troubles using webpack 5 with typescript because plugins stil have this package as a dependency.

Reply:
How will this affect people not using webpack 5? Does that even matter?

Reply:
Think you need to make a decision if you want to typecheck the webpack 5 code or the webpack 4 code. if you want to keep type checking webpack 4 code you should move this to dev dependencies so webpack 5 users won't have conflicts. 

But i'm not totally sure about this. In my recent projects I only supported webpack 5 from a certain version and not both at the same time.